Services/Administration/OJS.Services.Administration.Business/Implementations/ProblemsBusinessService.cs

```diff
@@ -176,6 +182,27 @@ public async Task<bool> UserHasProblemPermissions(int problemId, string? userId,
             return await this.contestsBusiness.UserHasContestPermissions(problem.ContestId, userId, isUserAdmin);
         }
 
+        public async Task ReevaluateProblemsByOrderBy(int contestId, Problem problem)
```

Comment:
The name is not correct. `ByOrderBy` indicates that OrderBy will be passed as an argument. Better name would be `ReevaluateProblemsOrder`. Same applies for ProblemGroups.